Doc/library/stdtypes.rst

```diff
@@ -4959,6 +4959,11 @@ All parameterized generics implement special read-only attributes.
       (~T,)
 
 
+   .. note::
+      A ``GenericAlias`` object with :class:`typing.ParamSpec` parameters may not
+      have correct ``__parameters__`` after substitution because
+      :class:`typing.ParamSpec` is intended primarily for static type checking.
```

Comment:
```python
from typing import ParamSpec, TypeVar, Generic

P = ParamSpec('P')
T = TypeVar('T')

class X(Generic[P]): ...

# Empty tuple, because T is nested inside that list
print(X[int, str, T].__parameters__)

```